src/components/scenes/Loans/LoanCreateConfirmationScene.tsx

```diff
@@ -39,7 +39,7 @@ const FEE_VOLATILITY_MULTIPLIER: { [network: string]: string } = {
 }
 
 interface Props {
-  navigation: NavigationProp<'loanCreateConfirmation'>
+  navigation: NavigationBase
```

Comment:
Revert using `NavigationBase` generic for scene components. The scene is the only component where `NavigationProp<Name>` makes sense because it's the type for that named scene. Where `NavigationBase` makes sense is in components which will take any ol' navigation, and wont expect scene specific information in the navigation object it gets.

Reply:
Address this for all the scenes you changeed.